File: app/src/main/java/io/zenandroid/greenfield/model/ImageListResponse.java
package io.zenandroid.greenfield.model;

import java.util.Date;
import java.util.List;

/**
 * Created by alex on 24/01/2018.
 */

public class ImageListResponse {
    private String title;
    private String link;
    private String description;
    private Date modified;
    private String generator;
    private List<Image> items;

    public List<Image> getItems() {
        return items;
    }

    public void setItems(List<Image> items) {
        this.items = items;
    }
}

File: app/src/main/java/io/zenandroid/greenfield/model/ImageListResponse.kt
package io.zenandroid.greenfield.model

import java.util.*

/**
 * Created by alex on 24/01/2018.
 */

data class ImageListResponse (
    private val title: String? = null,
    private val link: String? = null,
    private val description: String? = null,
    private val modified: Date? = null,
    private val generator: String? = null,
    var items: List<Image>? = null
)
